packages/services/storage/src/index.ts

```diff
@@ -4054,6 +4058,28 @@ const schemaCheckSQLFields = sql`
   , "manual_approval_user_id" as "manualApprovalUserId"
 `;
 
+const schemaCheckSQLFieldsFallback = sql`
+  "sc"."id"
+  , to_json("sc"."created_at") as "createdAt"
+  , to_json("sc"."updated_at") as "updatedAt"
+  , "sc"."schema_sdl" as "schemaSDL"
+  , "service_name" as "serviceName"
+  , "meta"
+  , "sc"."target_id" as "targetId"
+  , "schema_version_id" as "schemaVersionId"
+  , "is_success" as "isSuccess"
+  , "sc"."schema_composition_errors" as "schemaCompositionErrors"
+  , "breaking_schema_changes" as "breakingSchemaChanges"
+  , "safe_schema_changes" as "safeSchemaChanges"
+  , "schema_policy_warnings" as "schemaPolicyWarnings"
+  , "schema_policy_errors" as "schemaPolicyErrors"
+  , coalesce("sc"."composite_schema_sdl", "sv"."composite_schema_sdl") as "compositeSchemaSDL"
+  , coalesce("sc"."supergraph_sdl", "sv"."supergraph_sdl") as "supergraphSDL"
```

Comment:
In case the `"sc"."composite_schema_sdl"` and `"sc"."supergraph_sdl"` is `null`, because `sc"."schema_composition_errors"` is non-null, `"compositeSchemaSDL"` and "supergraphSDL" need to be `null`.

Reply:
I think this thing is handled by zod (supergraphSchemaSDL is declared as null there on partialFailedSchemaCheckModel)

Reply:
I don't see why this would be handled by zod. If zod only validates, it does not transform the data.

Reply:
added transformation